docs/src/development/faq.md

```diff
@@ -1,177 +0,0 @@
-These are mounted on a distributed file system (which is transparent for you).
-When you backup your environment, they are backed up as well.
-When you create a new staging environment,
-these mounts are cloned with the rest of your data.
-
-## What happens if I push a local branch to my project?
-
-If you push a local branch that you created with Git,
-you create what's called an [inactive environment](../other/glossary.md#inactive-environment).
-
-This means there aren't any services attached to this branch.
-
-You are able to convert an `inactive environment` into an `active environment`
-and vice versa back from the environment configuration page
-or using the CLI with `platform environment:activate`.
-
-## How does my live site scale?
-
-Your production environment gets a pool of resources based on your plan size,
-which is then split up between the applications and services you have defined.
-(For example, PHP 40%, MySQL 30%, Redis 10%, Solr 20%).
-
-To increase the pool of CPU and RAM that gets split among all of the containers, upgrade your plan.
-Your containers then have additional resources as soon as your site is redeployed.
-
-If you're expecting more traffic than usual, upgrade your plan ahead of time.
-You can downgrade your plan after the traffic surge to save resources
-and you only pay for the time you use the larger plan.
-
-All containers on development plans are "small" containers.
-
-See configuration options for [apps](../create-apps/app-reference.md#sizes)
-and [services](../add-services/_index.md#size).
-
-## What exactly am I accessing with SSH?
-
-The `platform ssh` command allows you to log into your application container (where your PHP app or Node app or Java app is running).
-It's a fully running Linux environment,
-but almost all of the disk is read-only, with the exception of mounts you have defined.
-
-## Can I edit a quick fix on a Platform environment without triggering a rebuild?
-
-No. Changes to the code can only be made through deploying new Git commits.
-That ensures that "hot patches" don't get lost in the net update, that all changes can be audited,
-and that if a security break-in happens the attacker still can't modify your application code.
-
-## What do I see when I push code?
-
-When you `git push` new code, Platform.sh rebuilds and redeploys the application.
-What shows on the command line is the output of your build process
-(such as Composer, pip, or Bundler plus your own build hook) followed by the deploy process.
-It ends with a description of what was just deployed and the URLs that are now active.
-
-To suppress the output, run `platform push -W`.
-The `-W` means `--no-wait` and disconnects the connection once the commits are pushed so that you can continue to use your local terminal.
-The exact same output is also available in the Console.
-
-## What Linux distribution is Platform.sh using?
-
-Platform.sh is built on Debian.
```

Comment:
:question: There's a bunch of stuff you took out that we don't need, for sure, which is always good. But this isn't present anywhere anymore. Do you think it might have value?

Reply:
Added it to [the introduction](https://docs.platform.sh/).